File: app/src/main/java/com/example/vkwall/data/model/GetWallByID/CopyHistoryWalls.java
package com.example.vkwall.data.model.GetWallByID;

import com.example.vkwall.data.model.ProfileWall.PostSource;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class CopyHistoryWalls {
    @SerializedName("id")
    private Integer id;
    @SerializedName("owner_id")
    private Integer ownerId;
    @SerializedName("from_id")
    private Integer fromId;
    @SerializedName("date")
    private Integer date;
    @SerializedName("post_type")
    private String postType;
    @SerializedName("text")
    private String text;
    @SerializedName("attachments")
    private List<AttachmentById> attachments;
    @SerializedName("post_source")
    private PostSource postSource;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public Integer getOwnerId() {
        return ownerId;
    }

    public void setOwnerId(Integer ownerId) {
        this.ownerId = ownerId;
    }

    public Integer getFromId() {
        return fromId;
    }

    public void setFromId(Integer fromId) {
        this.fromId = fromId;
    }

    public Integer getDate() {
        return date;
    }

    public void setDate(Integer date) {
        this.date = date;
    }

    public String getPostType() {
        return postType;
    }

    public void setPostType(String postType) {
        this.postType = postType;
    }

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }

    public List<AttachmentById> getAttachments() {
        return attachments;
    }

    public void setAttachments(List<AttachmentById> attachments) {
        this.attachments = attachments;
    }

    public PostSource getPostSource() {
        return postSource;
    }

    public void setPostSource(PostSource postSource) {
        this.postSource = postSource;
    }
}

File: app/src/main/java/com/example/vkwall/data/model/GetWallByID/CopyHistoryWalls.kt
package com.example.vkwall.data.model.GetWallByID

import com.example.vkwall.data.model.ProfileWall.Wall.PostSource
import com.google.gson.annotations.SerializedName

class CopyHistoryWalls {
    @SerializedName("id")
    var id: Int? = null
    @SerializedName("owner_id")
    var ownerId: Int? = null
    @SerializedName("from_id")
    var fromId: Int? = null
    @SerializedName("date")
    var date: Int? = null
    @SerializedName("post_type")
    var postType: String? = null
    @SerializedName("text")
    var text: String? = null
    @SerializedName("attachments")
    var attachments: List<AttachmentById>? = null
    @SerializedName("post_source")
    var postSource: PostSource? = null
}
